From: Wei Yongjun <weiyongjun1@huawei.com>
Date: Thu, 2 Jul 2020 16:07:50 +0800
Subject: thunderbolt: Fix old style declaration warning
Git-commit: ef7e12078ab832c72315adcfa05e7a9498a5e109
Patch-mainline: v5.9-rc1
References: jsc#SLE-14130

Fix gcc build warning:

drivers/thunderbolt/quirks.c:21:1: warning:
 'static' is not at beginning of declaration [-Wold-style-declaration]
   21 | const static struct tb_quirk tb_quirks[] = {
      | ^~~~~

Reported-by: Hulk Robot <hulkci@huawei.com>
Signed-off-by: Wei Yongjun <weiyongjun1@huawei.com>
Signed-off-by: Mika Westerberg <mika.westerberg@linux.intel.com>
Acked-by: Nicolas Saenz Julienne <nsaenzjulienne@suse.de>
---
 drivers/thunderbolt/quirks.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/thunderbolt/quirks.c b/drivers/thunderbolt/quirks.c
index 0525f59220ae..7eac3e0f90a2 100644
--- a/drivers/thunderbolt/quirks.c
+++ b/drivers/thunderbolt/quirks.c
@@ -18,7 +18,7 @@ struct tb_quirk {
 	void (*hook)(struct tb_switch *sw);
 };
 
-const static struct tb_quirk tb_quirks[] = {
+static const struct tb_quirk tb_quirks[] = {
 	/* Dell WD19TB supports self-authentication on unplug */
 	{ 0x00d4, 0xb070, quirk_force_power_link },
 };

